src/components/top-bar/top-bar.stories.tsx

```diff
@@ -1,124 +1,131 @@
 import { figmaDesignUrls } from '#src/storybook/figma/index'
 import type { Meta, StoryObj } from '@storybook/react'
-import { ElAvatar } from '../avatar'
-import { AvatarButton, ElAvatarButton } from '../avatar-button'
+import { AvatarButton } from '../avatar-button'
 import { elIcon } from '../button'
 import { CSSContainerQuery } from '../container-query/container-query'
 import { Icon } from '../icon'
-import { ElMenu, ElMenuItemAnchor, ElMenuItemButton, ElMenuList, ElMenuPopover, Menu } from '../menu'
-import { ElNavDropdownButton, NavDropdownButton } from '../nav-dropdown-button'
-import { ElButtonNavIconItem, NavIconItem } from '../nav-icon-item'
-import { ElNavItemAnchor, ElNavItemLabelContainer, NavItem } from '../nav-item'
+import { Menu } from '../menu'
+import { NavDropdownButton } from '../nav-dropdown-button'
+import { NavIconItem } from '../nav-icon-item'
+import { NavItem } from '../nav-item'
 import { NavSearchButton } from '../nav-search-button/nav-search-button'
-import {
-  ElNavSearchButton,
-  ElNavSearchButtonContainer,
-  ElNavSearchButtonIcon,
-  ElNavSearchButtonPlaceholder,
-  ElNavSearchButtonShortcutText,
-} from '../nav-search-button/styles'
 import { ReapitLogo } from '../reapit-logo'
-import reapitLogoIconUrl from '../reapit-logo/icons/brand-reapit.svg'
 import MenuIcon from './icons/menu-icon.svg?react'
-import {
-  ElTopBar,
-  ElTopBarLogo,
-  ElTopBarMainNav,
-  ElTopBarMobileNav,
-  ElTopBarProfile,
-  ElTopBarSearch,
-  ElTopBarSecondaryNav,
-  elTopBarMenuPopover,
-} from './styles'
+import { elTopBarMenuPopover } from './styles'
 import { TopBar } from './top-bar'
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
+console.log(INITIAL_VIEWPORTS)
+
+const viewports: typeof INITIAL_VIEWPORTS = {
```

Comment:
need to add additionals because storybook's provided viewport doesn't have widescreen .etc